fda-model/alternatives/Alternative.cs

```diff
@@ -25,6 +28,12 @@ public Alternative()
 
         #region Methods 
         /// <summary>
+        /// TODO TODO TODO TODO
+        /// This method is so god awfully complicated because we do not compute at the structure level 
```

Comment:
Maybe we need to make a GitHub discussion soon about why we don't compute at a structure level. I know we've had this conversation before, but I can't quite remember